spec/services/session_encryptor_spec.rb

```diff
@@ -21,16 +21,19 @@
     expect(encryptor2.load(encrypted_text)).to eq(payload)
   end
 
-  it 'does not modify the cek when failing to decrypt a payload encrypted with an old key' do
+  it 'does not modify the user access key when decrypting a payload encrypted with an old key' do
     old_encryptor = SessionEncryptor.new
     old_encryptor.user_access_key.unlock(Pii::Cipher.random_key)
```

Comment:
Hmm, this is actually broken now that `#user_access_key` is not longer a thing. I'll have to find a workaround

Reply:
Fixed. It doesn't really make sense to try to unlock the key now that the keys are effectively read-only and non-deterministic.